### PR TITLE
Adding a new page to aid users to transform from WUM/In-place to Updates

### PR DESCRIPTION
**Pull request by chamindi-a**
The issues addressed are :
How do I migrate from WUM and In-place to Updates 2.0?
How can a user merge the *WUM Diff* manually to get updates using WSO2 Updates 2.0?
Is it mandatory to have a configuration management tool configured or can we take the Updates manually?
what’s are the correct steps to apply the same Update level to all the nodes in the deployment if the user has not configured a configuration management tool?
If nodes are started with product profiles, how to take the manual Updates?
How to use Configuration Management Tool to get WUM updates?
A customer who runs on a containerized deployment, How to differentiate docker images created with new updates from WSO2 Docker Repository?

### CODE REVIEW

File: en/docs/updates/migrating-to-updates2.0.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    The diff ZIP is created in the location from where you executed the diff command.
+<br>
+3. Now your current pack is congruent with WSO2 Update 2.0, thus you are able to run the [Updates tool](../../updates/update-tool/) and receive WSO2 Updates.<br>
+4. Test the updated product distribution and deploy it in your production environment.
+
+### Is it mandatory to have a configuration management tool configured or can we take the Updates manually?
+It is not mandatory to have a configuration management tool configured, but having a configuration management tool makes the continuous updates process easy, 
+and it aids configuration with less possibility of human error.
+
+### what’s are the correct steps to apply the same Update level to all the nodes in the deployment if the user has not configured a configuration management tool?
```
Comment (Savidude) [inline en/docs/updates/migrating-to-updates2.0.md:24]: Shall we change this to `What are the correct steps to apply the same Update level to all the nodes in the deployment if a configuration management tool is not being used?`?

File: en/docs/updates/migrating-to-updates2.0.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+In some situations' a Configuration Management Tool will not be configured, however WSO2 recommends the use of a Configuration Management Tool to mitigate human errors 
+as the manual intervention may lead to some flaws. <br>
+Steps to apply the same update level across all node: <br>
+1. Apply the update to a pre-production environment.<br>
```
Comment (Savidude) [inline en/docs/updates/migrating-to-updates2.0.md:28]: Shall we change this to something like `In a pre-production environment, update to the required update level using the --level flag (eg: ./wso2update_linux --level 10)`

File: en/docs/updates/migrating-to-updates2.0.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+2. Run all testcases and verify the update in that environment.<br>
+3. Propagate the updates to production environment (using rolling updates, Canary or Blue-green update methods).
+
+### If nodes are started with product profiles, how to take the manual Updates?
```
Comment (Savidude) [inline en/docs/updates/migrating-to-updates2.0.md:32]: Shall we remove this? We can develop a fix for this in the update tool. Similar to what we have done with in-place.

Comment (chamindi-a) [reply en/docs/updates/migrating-to-updates2.0.md:32]: leaving the comment unresolved as the support team need the information added here

**Review by VimukthiPerera — APPROVED** [review]

**Review by Savidude — APPROVED** [review]

File: en/docs/updates/migrating-to-updates2.0.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+### How do I migrate from WUM and In-place to Updates 2.0?
+
+* Update the current product pack to the latest update level using WUM or in-place tool. This will make Update tool
```
Comment (jsdjayanga) [inline en/docs/updates/migrating-to-updates2.0.md:3]: This will make *new* Update tool

File: en/docs/updates/migrating-to-updates2.0.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
+### How do I migrate from WUM and In-place to Updates 2.0?
+
+* Update the current product pack to the latest update level using WUM or in-place tool. This will make the new Update tool
+  client available inside `<product_home>/bin` directory.
+* Run the [Update commands](../../updates/update-tool/) on top of the update tool. This will
```
Comment (jsdjayanga) [inline en/docs/updates/migrating-to-updates2.0.md:5]: shouldn't this be "on top of the product pack"?

File: en/docs/updates/migrating-to-updates2.0.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+2. Run all testcases and verify the update in that environment.<br>
+3. Propagate the same updated product pack to other and production environments(using rolling updates, Canary or Blue-green update methods).
+
+### If nodes are started with product profiles, how to take updates?
+First take the update of the base product and thereafter distribute the updated pack to other deployment environments. 
+Then, run different profiles on those environments as needed.
+
+### How to migrate from WUM to the new Updates model if a Configuration Management Tool is being used ?
+If a Configuration Management Tool is being used, follow the steps below to convert to the new updates model.<br>
+The respective product pack lies in the `files/packs` directory, to migrate to updates 2.0 seamlessly you have to make both the product pack, and the `/scripts/update.sh` file compatible with Updates 2.0.
```
Comment (VimukthiPerera) [inline en/docs/updates/migrating-to-updates2.0.md:49]: `/scripts/update.sh` should be `scripts/update.sh`

File: en/docs/updates/migrating-to-updates2.0.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+<br>
+- To update the **product pack** - Go to` scripts` directory and run the `./update.sh` script.<br>
+- To update **/scripts/update.sh** file - Checkout the latest [v3.2.0.x](https://github.com/wso2/ansible-apim/tags) tags, in this example WSO2 APIM.
```
Comment (VimukthiPerera) [inline en/docs/updates/migrating-to-updates2.0.md:52]: Same as above.

File: en/docs/updates/migrating-to-updates2.0.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+### Is it mandatory to have a configuration management tool configured or can we take the Updates manually to the environments?
+It is not mandatory to have a configuration management tool configured, but having a configuration management tool makes the continuous updates process easy, 
+and it aids configuration with less possibility of human errors.
+
+### How the keystore and truststore configurations are incorporated with a configuration management tool?
+Refer [APIM Ansible repository Documentation](https://github.com/wso2/ansible-apim/tree/3.2.x#including-custom-keystore-and-truststore) to get an insight on customizing keystore and truststore configurations that are integrated with a configuration management tool.
+
+### How to persist customizations done on the servers during each update?
+1. If there are .jar or UI customization those changes should be done in configuration management server.Those changes should also be pushed to your forked repository, for future use.<br>
```
Comment (VimukthiPerera) [inline en/docs/updates/migrating-to-updates2.0.md:31]: Missing space `server.Those`

File: en/docs/updates/migrating-to-updates2.0.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+2. The customized files should be in the sub-directories of the `/files` directory. (Particularly, all .jar files and UI customizations should be added to the `misc` folder.)<br>
```
Comment (VimukthiPerera) [inline en/docs/updates/migrating-to-updates2.0.md:32]: `files` instead of `/files`

**Pull request merged**